src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php

```diff
@@ -55,15 +55,17 @@ public function __construct(Connection $conn)
      */
     public function loadTokenBySeries(string $series)
     {
-        // the alias for lastUsed works around case insensitivity in PostgreSQL
```

Comment:
This was introduced by https://github.com/symfony/symfony/pull/27314. The proposed patch also seems to handle postgres issue too.